src/System.Reflection/ref/System.Reflection.cs

```diff
@@ -14,7 +14,7 @@ public sealed partial class AmbiguousMatchException : System.Exception
         public AmbiguousMatchException(string message) { }
         public AmbiguousMatchException(string message, System.Exception inner) { }
     }
-    public abstract partial class Assembly
+    public abstract partial class Assembly : ICustomAttributeProvider
```

Comment:
It is best if we fully qualify this type here in the other places it is referenced. While it isn't strictly needed it will help reduce noise later when we generate this file again with a tool.